Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - ClothingOuterHardsuitSpatio
+        - WeaponLaserSvalinn
+        - TechnologyDisk
+        - ClothingHeadHelmetTemplar
+        - SheetUranium
+        - ClothingNeckCloakVoid
+        - BoxHandcuff
+        - ResearchDisk5000
+        - ResearchDisk
+        - WeaponRevolverDeckardNonlethal
+        - ToySword
+        - ClothingShoesBootsSpeed
+        - PinpointerUniversal
+      chance: 1
+      rarePrototypes:
+        - ClothingBeltSheathFilled
+        - WeaponShotgunKammerer
+        - WeaponForceGun
+        - WeaponParticleDecelerator
+        - ClothingOuterArmorPodWars
+        - ClothingHeadHelmetPodWars
+        - ClockworkShield
+        - MirrorShield
+        - BedsheetUSA
+        - HypoMini
+        - HypoMiniLimitedEdition
+        - RCDRecharging
+        - ClothingShoesBootsSpeed
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - ClothingBackpackHolding
+        - ClothingBackpackMessengerHolding
+        - ClothingBackpackDuffelHolding
+        - ClothingShoesBootsMagAdv
+        - AdvMopItem
+        - RGBStaff
+        - WeaponAdvancedLaser
+        - AmmoTechFabCircuitboard
+        - PlushieArachind
+        - ClothingMaskArcadia
+        - ClothingBeltChiefEngineerFilled
+        - CowToolboxFilled
+        - ClothingShoesBootsMagCombatFilled
+        - ClothingShoesBootsMagGaloshes
+        - Hypospray
+        - Katana
+        - WeaponLaserCannon
+        - MuteToxinChemistryBottle
+        - ClothingOuterHardsuitCap
+        - ClothingShoesBootsMagBlinding
+        - TurboItemRechargerCircuitboard
+        - WeaponRevolverDeckard
+        - MedicatedSuture
+        - FoodPizzaArnold
+        - PinpointerUniversal
+        - Claymore
+      rareChance: 0.3
+
+- type: entity
+  id: RandomRarelegalLoot
+  name: Random Rare Legal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetUSA
+    - type: RandomSpawner
+      prototypes:
+        - ClothingBeltSheathFilled
+        - SpaceCash10000
+        - WeaponAntiqueLaser
+        - WeaponForceGun
+        - PenCentcom
+        - ClothingOuterArmorPodWars
+        - ClothingHeadHelmetPodWars
+        - ClothingOuterCoatSpaceAsshole
+        - UnholyHalberd
+        - ClockworkShield
+        - MirrorShield
+        - BedsheetUSA
+        - HypoMini
+        - HypoMiniLimitedEdition
+        - ClothingBackpackHolding
+        - ClothingShoesBootsMagAdv
+        - OreBagOfHolding
+        - ClothingShoesBootsSpeed
+        - ClothingBackpackDuffelHolding
+        - ClothingBackpackMessengerHolding
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - AdvMopItem
+        - HandheldCrewMonitor
+        - WeaponAdvancedLaser
+        - VariedXenoArtifactItem
+        - ClothingMaskArcadia
+        - ClothingShoesBootsMagCombatFilled
+        - WeaponLaserCannon
+        - ClothingEyesHudMedSec
```

Comment:
Don't include sec items in a legal list.

Reply:
made illegal